src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
+const initialState = {
+  booksArray: [],
+};
+
+export default function bookReducer(state = initialState, action) {
+  switch (action.types) {
+    case ADD:
+      return {
+        ...state,
+        booksArray: [...state.booksArray, action],
+      };
+    case REMOVE:
+      return {
+        ...state,
+        booksArray: [...state.booksArray, action],
+
+      };
```

Comment:
- [ ] Please let's make sure that the particular book is removed from the `booksArray` in case the action type is `REMOVE`. You may use the ES6 filter() method to create a new array, which will not contain the book you want to remove from the store.